test/case/convention/openshift/golang-osd-operator/01_generated_files_checker

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -x
```

Comment:
Remove `-x`. Set `BOILERPLATE_SET_X` to enable it on a per-run basis.

Also, use `-e` as you convinced me we should :P